lib/ask/o_auth_token.ex

```diff
@@ -37,4 +37,15 @@ defmodule Ask.OAuthToken do
   def access_token(token) do
     Poison.Decode.decode(token.access_token, as: %OAuth2.AccessToken{})
   end
+
+  def about_to_expire?(token) do
```

Comment:
Just asking here... Why to change the `Timex` implementation of 2 lines for this one?

Reply:
The Timex library isn't deprecated, but Elixir added lots of support for dates, times and timezones right into the stdlib, and Timex mostly delegates to these types, now. The [library maintainers](https://github.com/bitwalker/timex#migrating-to-timex-3x) state that themselves.